bedrock/firefox/urls.py

```diff
@@ -32,7 +32,7 @@
     url(r'^firefox/(?:%s/)?(?:%s/)?all/$' % (platform_re, channel_re),
         views.all_downloads, name='firefox.all'),
     page('firefox/accounts', 'firefox/accounts.html'),
-    page('firefox/channel', 'firefox/channel.html'),
+    page('firefox/channel', 'firefox/channel.html', OLDIEJS=True),
```

Comment:
One pain point of this approach will be adding this context var for *all* pages that haven't yet been tested without JS in old IE. I went this way so we could remove code as we went along, as opposed to adding code to opt pages in to the new, minimal JS way. Happy to hear some opposing thoughts here.